atomic_reactor/plugins/pre_add_yum_repo_by_url.py

```diff
@@ -94,7 +94,7 @@ class AddYumRepoByUrlPlugin(PreBuildPlugin):
     key = "add_yum_repo_by_url"
     is_allowed_to_fail = False
 
-    def __init__(self, tasker, workflow, repourls, inject_proxy=None):
+    def __init__(self, tasker, workflow, repourls=None, inject_proxy=None):
```

Comment:
Curious as to what error this was causing.

I'd think that calling this plugin without `repourls` is not correct.

Reply:
PluginsConfiguration currently doesn't remove add_yum_repo_by_url if there are valid yum_repourls values. I guess this could be removed and PluginsConfiguration could remove add_yum_repo_by_url - either is good.

Reply:
This is probably fine.

Reply:
Leaving alone for now.